### PR TITLE
fix: available space of source code

### LINKED ISSUE #1133: display of source code doesn't make use of available space
Example: https://deno.land/x/wasi@7fc9454/mod.ts#L571-L596
Screen Shot 2020-06-15 at 12 51 07 PM

A combination of borders and large font size make it so I can't see this reasonably sized code snippit on my laptop.

[Github](https://github.com/caspervonb/deno-wasi/blob/7fc945435e0a61f0f33b3a9f42100ba43fa2e8ac/mod.ts#L571-L596) isn't great either - but at least I can view this function:
Screen Shot 2020-06-15 at 12 52 30 PM

### PR DESCRIPTION
**Pull request by rjoydip**
Closes #1133

### CODE REVIEW

**Comment by lucacasonato** [comment]
I am not fully happy with this PR - reducing the text size makes sense for large code blocks, but reduces legibility significantly, and doesn't look great for the small code snippets in the manual and on the homepage. I think the code snippets should be `text-sm`. The registry can stay `text-xs`.

File: components/CodeBlock.tsx [hunk 1]
```diff
@@ -92,19 +94,14 @@ export const RawCodeBlock = ({
...
             (language === "bash" || language === "shell") && (
               <code className="pr-2 sm:pr-3">
-                <div className="text-gray-400 token-line text-right select-none">
-                  $
-                </div>
+                <div className={codeDivClassNames}>$</div>
               </code>
             )}
           {tokens.length > 1 && !disablePrefixes && (
-            <code className="pr-2 sm:pr-3">
```
[the remark this answers is not in the document]
Comment (rjoydip) [reply components/CodeBlock.tsx:101]: @littletof Could you tell where is it?

[the remark this answers is not in the document]
Comment (littletof) [reply components/CodeBlock.tsx:101]: The problem comes up on any repo on deno.land that has code blocks in its readme eg: https://deno.land/x/oak

[the remark this answers is not in the document]
Comment (rjoydip) [reply components/CodeBlock.tsx:101]: PR https://github.com/denoland/deno_website2/pull/1176 revert back the padding

**Please check deployment and confirm.**

In the readme of deno third party modules https://deno-website2-hgxvfyj0o.vercel.app/x/oak and in the manual: https://deno-website2-hgxvfyj0o.vercel.app/manual/getting_started/first_steps#reading-a-file

[the remark this answers is not in the document]
Comment (littletof) [reply components/CodeBlock.tsx:101]: @rjoydip LGTM

**Pull request merged**